src/app/share/views/SharedLinksView/SharedView.tsx

```diff
@@ -121,7 +121,14 @@ export default function SharedView(): JSX.Element {
 
   useEffect(() => {
     if (!currentFolderId && !isShareDialogOpen && !folderUUID) {
-      setTimeout(fetchRootFolders, 200);
+      setTimeout(() => {
+        setPage[0];
+        setShareItems([]);
+        setHasMoreRootFolders(true);
+        setHasMoreFolders(true);
+        setHasMoreItems(true);
+        fetchRootFolders();
```

Comment:
If all of these it is for reset the Shared view state, add it to a function and reuse it in all places that is need it.

Reply:
Done